src/plugin.rs

```diff
@@ -183,18 +183,19 @@ fn shared_access_ui<T>(
     };
 
     let window_data = inspector_windows.window_data::<T>();
-    let ctx = egui_context.ctx_for_window(window_data.window_id);
+    let [window_ctx, primary_ctx] =
```

Comment:
What is the reason for using the primary context to show the window? Wouldn't this result in the inspector window being shown on the primary window even if the user requested it to be on another one?

Reply:
I'm not sure what the initial code did, but I just translated it to the new version of `bevy_egui` to make it compile.
I now pass `primary_ctx` instead of `egui_context.ctx()`, which also fetched the context of the primary window. Does this make sense?

Reply:
Oh right, then the bug was already present before :grimace:. Both should be using the context of the window.
I'll merge this and then fix the usage of the wrong egui context.